src/extensions/sessionrecording.ts

```diff
@@ -127,6 +82,7 @@ export class SessionRecording {
     rrwebRecord: rrwebRecord | undefined
     recorderVersion?: string
     isIdle = false
+    rateLimiter?: MutationRateLimiter
```

Comment:
we're adding a `rateLimiter` in another PR too 🤔 

maybe this should be named explicitly `mutationRateLimiter`